Reference/combat_locations_mapdata.json

```diff
@@ -0,0 +1,882 @@
+[
+  {
+    "featureId": "dungeon-corrupted-decrepit-sewers",
+    "categoryId": "wynntils:content:dungeon",
```

Comment:
Maybe we should even have a `wynntils:content:dungeon:corrupted` category?

Reply:
I've thought about it, but I am not sure if it was worth the edge case. I can do it, I guess.

Reply:
I think it would be nice to have it, so if it is not too much work I'd appreciate if you could do it.